### PR TITLE
Writing a tutorial on how to create a SC

### PR DESCRIPTION
**Pull request by yvan-sraka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,51 @@
...
-## Scripts
+## Usage
 
-You can run scripts described in the package.json with `yarn run {script_name}`, if you're not confident with the package.json, take a minute to look on the [yarn documentation](https://classic.yarnpkg.com/lang/en/docs/cli/run/).
+You can run scripts described in the package.json with `yarn run {script_name}`, if you're not confident with the package.json, take a minute to look at the [yarn documentation](https://classic.yarnpkg.com/lang/en/docs/cli/run/).
 
-I'll describe the differents embeded scripts in this section and redirect to severals documentations if needed.
+I'll describe the different embedded scripts in this section and redirect to several documentations if required.
 
-- **helloworld**: generate a .wasm file in a `build/` directory `src/helloworld.ts`. This is the helloworld smart contract example. If you look at this command line, you can see that we use the asc binary, this is the AssemblyScript compiler. Here is [the documentation](https://www.assemblyscript.org/introduction.html) if you want to know more.
```
Comment (yvan-sraka) [inline README.md:24]: @adrien-zinger my will is still to simplify this bunch of command into few, as described here https://github.com/massalabs/massa-sc-template/issues/3 using `massa-sc-scripts`!

File: README.md [hunk 2]
```diff
@@ -1,46 +1,51 @@
...
+- `exec`: run `buildAll` and then execute the smart contract on a mock with the `massa-sc-test` binary. Look at the mock repository [here](https://github.com/massalabs/massa-sc-tester) for more information about mocking the network.
+- `buildTest`: Build `mytest.ts`
+- `execTest`: Exec in the mocked network `mytest.wasm`
 
 ## Mocked ledger
 
-Once you ran the **exec** script, you should see a new file in the directory named `ledger.json`. This file represent the local state of the ledger for your test. You can modify it manually (obviously, carrefully) to look at the state of the ledger after each execution.
+Once you ran the `exec` script, you should see a new file in the directory named `ledger.json`. This file represent the local state of the ledger for your test. You can modify it manually (obviously, carefully) to look at the state of the ledger after each execution.
 
 > This mock doesn't manage like the real massa network backups when an execution failed for the moment. If you want to keep immutable the ledger make a copy before locally.
```
Comment (qdrn) [inline README.md:47]: > If you want to keep immutable the ledger make a copy before locally.
I'm not sure I understand what this sentence means. Maybe you can reformulate ?

**Review by qdrn — APPROVED** [review]
Maybe you could add an example (how do I know that everything worked?) and show how to mock the network.

**Comment by adrien-zinger** [comment]
> Maybe you could add an example (how do I know that everything worked?) and show how to mock the network.

Note that you've got massa-sc-tester that use this lib and totally mock the network

File: README.md (hunk 2 above)
Comment (adrien-zinger) [reply README.md]: Updated :+1: 

I simply wanted to explain that the ledger test has no backup behavior in the case where a smart contract exit with an error.

**Review by adrien-zinger — APPROVED** [review]

**Comment by adrien-zinger** [comment]
bors merge

**Pull request merged**